c/type1/source/type1.c

```diff
@@ -526,7 +530,9 @@ int main(int argc, char *argv[]) {
                 type1(fp1,fp2);
 				fclose(fp1);
 				fclose(fp2);
-	} else
-		usage();
-	return 0;
+    } else
```

Comment:
Codacy sort of hints at the problem here, which is that you need to add some braces after your `else` and between the `return` statements, i.e.:
```c
    } else {
        usage();
        return 1;
    }
    return 0;
}
```

Reply:
👍 in that case I need to add braces to `detype1.c` as well.